pom.xml

```diff
@@ -102,24 +102,45 @@
       <artifactId>devon4j-basic</artifactId>
       <version>${devon4j.version}</version>
     </dependency>
+    
     <dependency>
       <groupId>com.devonfw.java.modules</groupId>
       <artifactId>devon4j-test</artifactId>
       <version>${devon4j.version}</version>
       <scope>test</scope>
     </dependency>
+    
     <dependency>
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
       <version>1.2.0</version>
       <scope>test</scope>
     </dependency>
+    
     <dependency>
       <groupId>org.codehaus.sonar-plugins.java</groupId>
       <artifactId>java-checks</artifactId>
       <version>3.2</version>
       <scope>test</scope>
     </dependency>
+    
+    <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
+      <version>1.3.2</version>
+    </dependency>
+    
+    <dependency>
+      <groupId>net.sf.m-m-m</groupId>
+      <artifactId>mmm-util-core</artifactId>
+      <version>8.7.0</version>
+    </dependency>
+    
+    <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-core</artifactId>
```

Comment:
Same here? Was it an accident that you added these dependencies in this PR?

Reply:
True, they are not used. I must have added them when I was trying a different approach for this rule. I will delete them.